Project/STM8S_StdPeriph_test/SDCC/Makefile

```diff
@@ -14,7 +14,7 @@
 #######
 
 # STM8 device (for supported devices see stm8s.h)
-DEVICE=STM8S103
```

Comment:
any reason to change the default device ?
I kind of like the 103 as it is the most available dev board on aliexpress ;-)